src/components/shared/Accordion.css

```diff
@@ -60,6 +60,6 @@
   height: 16px;
 }
 
-accordion .header-buttons button::-moz-focus-inner {        
+.accordion .header-buttons button::-moz-focus-inner {
```

Comment:
I also added a `.` here which is what I believe is needed and was just missed from copy / pasta